### DISCUSSION TITLE
Search file bằng C hay Bash thì nhanh hơn?

### DISCUSSION THREAD

**Discussion by chungngx** [codingreshapefuture/Community#23] [Q&A] [answered]
Search file bằng C hay Bash thì nhanh hơn?

**Comment by lucthienphong1120** [comment, marked as answer]
bash hay zsh cũng gọi các program để thực hiện “lệnh” của bạn thôi. nên gọi các program để tìm kiếm như find sẽ nhanh hơn rất nhiều. vì những program được tối ưu rất kinh khủng rồi. còn C bạn viết lại thì sẽ chậm hơn nhiều khi phải xử lý một lượng lớn dữ liệu

tất nhiên, không phải mọi program đều sẽ được tối ưu đàng hoàng, nên ưu tiên tìm những program thay thế như find -> locate, không tìm được thì hãy tự viết bằng C.

**Reply by chungngx** [reply]
well knowledge
